File: webservices/CORE/Comercio.php
<?php


class Comercio
{
    public $idcomercio;
    public $codigo;
    public $nombre_comercial;
    public $estado;
    public $direccion,$idciudad,$correo,$telefono,$idusuario;

    /**
     * Comercio constructor.
     * @param $idcomercio
     * @param $codigo
     * @param $nombre_comercial
     * @param $estado
     * @param $direccion
     * @param $idciudad
     * @param $correo
     * @param $telefono
     * @param $idusuario
     */
    public function __construct($idcomercio, $codigo, $nombre_comercial, $estado, $direccion, $idciudad, $correo, $telefono, $idusuario)
    {
        $this->idcomercio = $idcomercio;
        $this->codigo = $codigo;
        $this->nombre_comercial = $nombre_comercial;
        $this->estado = $estado;
        $this->direccion = $direccion;
        $this->idciudad = $idciudad;
        $this->correo = $correo;
        $this->telefono = $telefono;
        $this->idusuario = $idusuario;
    }

    /**
     * @return mixed
     */
    public function getIdcomercio()
    {
        return $this->idcomercio;
    }

    /**
     * @param mixed $idcomercio
     */
    public function setIdcomercio($idcomercio)
    {
        $this->idcomercio = $idcomercio;
    }

    /**
     * @return mixed
     */
    public function getCodigo()
    {
        return $this->codigo;
    }

    /**
     * @param mixed $codigo
     */
    public function setCodigo($codigo)
    {
        $this->codigo = $codigo;
    }

    /**
     * @return mixed
     */
    public function getNombreComercial()
    {
        return $this->nombre_comercial;
    }

    /**
     * @param mixed $nombre_comercial
     */
    public function setNombreComercial($nombre_comercial)
    {
        $this->nombre_comercial = $nombre_comercial;
    }

    /**
     * @return mixed
     */
    public function getEstado()
    {
        return $this->estado;
    }

    /**
     * @param mixed $estado
     */
    public function setEstado($estado)
    {
        $this->estado = $estado;
    }

    /**
     * @return mixed
     */
    public function getDireccion()
    {
        return $this->direccion;
    }

    /**
     * @param mixed $direccion
     */
    public function setDireccion($direccion)
    {
        $this->direccion = $direccion;
    }

    /**
     * @return mixed
     */
    public function getIdciudad()
    {
        return $this->idciudad;
    }

    /**
     * @param mixed $idciudad
     */
    public function setIdciudad($idciudad)
    {
        $this->idciudad = $idciudad;
    }

    /**
     * @return mixed
     */
    public function getCorreo()
    {
        return $this->correo;
    }

    /**
     * @param mixed $correo
     */
    public function setCorreo($correo)
    {
        $this->correo = $correo;
    }

    /**
     * @return mixed
     */
    public function getTelefono()
    {
        return $this->telefono;
    }

    /**
     * @param mixed $telefono
     */
    public function setTelefono($telefono)
    {
        $this->telefono = $telefono;
    }

    /**
     * @return mixed
     */
    public function getIdusuario()
    {
        return $this->idusuario;
    }

    /**
     * @param mixed $idusuario
     */
    public function setIdusuario($idusuario)
    {
        $this->idusuario = $idusuario;
    }



}

File: Facturador/app/src/main/java/com/example/facturador/Activity/Model/Comercio.java
package com.example.facturador.Activity.Model;

import java.io.Serializable;

public class Comercio implements Serializable
{
    private int idcomercio;
    private String codigo;
    private String nombre_comercial;
    private String direccion;
    private String correo;
    private String telefono;
    private String estado;
    private int idusuario;
    private int idciudad;
    private int id_base;

    public String getEstado() {
        return estado;
    }

    public void setEstado(String estado) {
        this.estado = estado;
    }

    public Comercio() {

    }

    public int getId_base() {
        return id_base;
    }

    public void setId_base(int id_base) {
        this.id_base = id_base;
    }

    public Comercio(int idcomercio, String codigo, String nombre_comercial, String direccion, String correo, String telefono, int idusuario, int idciudad) {
        this.idcomercio = idcomercio;
        this.codigo = codigo;
        this.nombre_comercial = nombre_comercial;
        this.direccion = direccion;
        this.correo = correo;
        this.telefono = telefono;
        this.idusuario = idusuario;
        this.idciudad = idciudad;
    }

    public int getIdcomercio() {
        return idcomercio;
    }

    public void setIdcomercio(int idcomercio) {
        this.idcomercio = idcomercio;
    }

    public String getCodigo() {
        return codigo;
    }

    public void setCodigo(String codigo) {
        this.codigo = codigo;
    }

    public String getNombre_comercial() {
        return nombre_comercial;
    }

    public void setNombre_comercial(String nombre_comercial) {
        this.nombre_comercial = nombre_comercial;
    }

    public String getDireccion() {
        return direccion;
    }

    public void setDireccion(String direccion) {
        this.direccion = direccion;
    }

    public String getCorreo() {
        return correo;
    }

    public void setCorreo(String correo) {
        this.correo = correo;
    }

    public String getTelefono() {
        return telefono;
    }

    public void setTelefono(String telefono) {
        this.telefono = telefono;
    }

    public int getIdusuario() {
        return idusuario;
    }

    public void setIdusuario(int idusuario) {
        this.idusuario = idusuario;
    }

    public int getIdciudad() {
        return idciudad;
    }

    public void setIdciudad(int idciudad) {
        this.idciudad = idciudad;
    }
}
